src/client/csfle/state_machine.rs

```diff
@@ -40,9 +40,9 @@ impl CryptExecutor {
         key_vault_namespace: Namespace,
         tls_options: Option<KmsProvidersTlsOptions>,
     ) -> Result<Self> {
-        let num_cpus = std::thread::available_parallelism()?.get();
+        // TODO(RUST-1492): Replace num_cpus with std::thread::available_parallelism.
```

Comment:
```suggestion
        // TODO RUST-1492: Replace num_cpus with std::thread::available_parallelism.
```
slight nit to help with grep-ability, since our existing TODOs don't have the parens.